assertj-refaster-rules/baseline-class-uniqueness.lock

```diff
@@ -2,384 +2,8 @@
 # Run ./gradlew checkClassUniqueness --write-locks to update this file
 
 ## runtimeClasspath
-[com.google.auto.service:auto-service-annotations, com.google.errorprone:error_prone_refaster]
+[com.google.auto.service:auto-service-annotations, com.google.errorprone:error_prone_core, com.google.errorprone:error_prone_refaster]
```

Comment:
We opt out of the uniqueness check in our other refaster checks, otherwise upgrades get blocked until humans can allow them through.